TombEngine/Game/items.cpp

```diff
@@ -42,16 +42,24 @@ constexpr auto ITEM_DEATH_TIMEOUT = 4 * FPS;
 
 BoundingBox ItemInfo::GetAabb() const
 {
-	return Geometry::GetBoundingBox(GetObb());
+	return Geometry::GetAabb(GetObb());
 }
 
 BoundingOrientedBox ItemInfo::GetObb() const
 {
 	auto frameData = GetFrameInterpData(*this);
 	if (frameData.Alpha == 0.0f)
-		return frameData.FramePtr0->BoundingBox.ToBoundingOrientedBox(Pose);
+		return BoundingOrientedBox(frameData.Keyframe0.Aabb.Center, frameData.Keyframe0.Aabb.Extents, Pose.Orientation.ToQuaternion());
```

Comment:
This return is **incorrect**. It returns local keyframe's bounding box center, not global center. You should add `Pose.Position.ToVector3()` to the first `BoundingOrientedBox` argument.